### PR TITLE
C3

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR add a C3 event specific page.

Dataset listing requires etalab/udata#55

### CODE REVIEW

File: udata_gouvfr/views.py [hunk 1]
```diff
@@ -72,6 +72,22 @@ def dataconnexions():
...
+    'etalab',
+    'ministere-de-l-ecologie-du-developpement-durable-et-de-l-energie',
+    'museum-national-dhistoire-naturelle',
+)
+
+
+@blueprint.route('/c3')
+def c3():
+    partners = Organization.objects(slug__in=C3_PARTNERS)
+    datasets = Dataset.objects(badges__kind=DATASET_BADGE_KINDS['c3'])
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: You should use the [newly introduced](https://github.com/etalab/udata/pull/55/files#diff-69d191c77db7ac001b315c1625049ee7R54) `c3` var

File: udata_gouvfr/theme/theme.py [hunk 2]
```diff
@@ -50,7 +50,11 @@ def __init__(self, label, page, **kwargs):
         nav.Item(_('Organizations'), 'organizations.list'),
     ]),
     nav.Item(_('Dashboard'), 'site.dashboard'),
-    nav.Item('Dataconnexions', 'gouvfr.dataconnexions'),
+    nav.Item(_('Events'), '#', url='#', items=[
+        nav.Item('Climate Change Challenge (C³)', 'gouvfr.c3'),
+        nav.Item('Dataconnexions', 'gouvfr.dataconnexions'),
```
Comment (davidbgk) [inline udata_gouvfr/theme/theme.py]: Do we really need to keep dataconnexions?

**Comment by davidbgk** [comment]
Please reduce the size of the image before merging :)

**Comment by davidbgk** [comment]
r+

File: udata_gouvfr/theme/theme.py (hunk 2 above)
Comment (noirbizarre) [inline udata_gouvfr/theme/theme.py]: Yep, there will be other editions, we will maybe split editions.

File: udata_gouvfr/views.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from flask import url_for, redirect
 
 from udata import theme
-from udata.models import Reuse, Organization
+from udata.models import Reuse, Organization, Dataset, DATASET_BADGE_KINDS
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: You can directly import `C3`

**Pull request merged**